src/index.ts

```diff
@@ -39,12 +39,12 @@ export const allTransforms = [
     reactRemoveStaticPropTypesMemberTransformFactoryFactory,
 ];
 
-export type TransformFactoryFactory = (typeChecker: ts.TypeChecker) => ts.TransformerFactory<ts.Node>;
+export type TransformFactoryFactory = (typeChecker: ts.TypeChecker) => ts.TransformerFactory<ts.SourceFile>;
```

Comment:
TypeScript will complain it if we use `<ts.Node>` here.

![image](https://user-images.githubusercontent.com/6076919/35179698-38053012-fdda-11e7-8eb5-d8925ff4c0ee.png)